crates/specs/src/args.rs

```diff
@@ -39,11 +39,24 @@ pub fn parse_args(values: Vec<&str>) -> Vec<u64> {
                             u64::from_le_bytes(data)
                         })
                         .collect::<Vec<u64>>()
-                }
+                },
+                "file" => {
```

Comment:
please update `single_public_arg` and README

Reply:
fixed in https://github.com/DelphinusLab/zkWasm/pull/225/commits/86be5b47c7e817887691ee7e5d35c226a6148968